src/compiler/prioritization/priority-levels.md

```diff
@@ -0,0 +1,96 @@
+
+- P-Critical
+- P-High
+- P-Medium
+- P-Low
+
+Each of these labels defines a strategy the team will adopt regarding:
+
+- The amount of focus a given issue will receive
+- How members of the community can get involved
+
+## Definitions
+
+### P-Critical
+
+A `P-Critical` issue is a potentially blocker issue.
+
+The Working Group will keep track of the number of such issues open at the same time (which shouldn't be more than 5), and how long such issues remain open.
+
+Examples of things we typically judge to be “critical” bugs:
+
+- Regressions where code that used to compile no longer does
+
+  - Mitigating conditions that may lower priority:
+    - If the code should never have compiled in the first place (but if the regression affects a large number of crates, this may indicate that we need a warning period)
+    - If the code in question is theoretical and considered unlikely to exist in the wild, or if it only exists in small, unmaintained packages that are not widely used
+  - If a regression has been in stable for a release or two (either because we are still awaiting a fix, or because the bug had laid dormant i.e. undetected), we typically lower the priority as well, because by that time, if the users have not raised a ruckus about the regression, that is a sign that it is inherently not a critical issue. Eg: [an issue that would have been P-Critical but ended up being P-High](https://rust-lang.zulipchat.com/#narrow/stream/227806-t-compiler.2Fwg-prioritization/topic/pre-meeting.20triage.202020-04-09.20.2354818)
+
+- Regressions where code still compiles but does something different than it used to do (dynamic semantics have changed)
+  - Mitigating conditions that may lower priority:
+    - If code uses feature that is explicitly not specified (e.g. `std::vec::Vec` docs state order in which it drops its elements is subject to change)
+- Feature-gated features accessible without a feature gate
+  - Mitigating conditions that may lower priority:
+    - If the pattern is VERY unlikely
+- Soundness holes where common code that should not compile actually does
+  - Mitigating conditions that may lower priority:
+    - Soundness holes that are difficult to trigger
+    - Soundness holes that have been around for a very long time may be critical, but typically require
+- Diagnostic regressions where the diagnostic is very common and the situation very confusing
+- ICEs for common scenarios or code patterns
+  - Mitigating conditions that may lower priority:
+    - If the code that triggers the ICE also triggers compilation errors, and those errors are emitted before the ICE
+    - If the code in question makes use of unstable features, particularly if the ICE requires a feature gate
+
+### P-High
+
+`P-High` issues are the ones the compiler team will focus on the most.
```

Comment:
`P-critical` are the ones that the compiler team focuses the most.

Maybe just remove this and start with `P-high` issues can be `P-critical` issues that have mitigating ...

Reply:
I think we can keep it, but start with a base definition and rearrange the wording. Suggestion:

### P-high

`P-high` issues are issues that need attention from the compiler team, but not to the point that they need to be discussed at every meeting. They can be `P-critical` issues that have a mitigating condition as defined above, or important issues that aren't deemed blockers.

Because there are too many `P-high` issues to fit in every compiler meeting, they should rather be handled *asynchronously* by the Prioritization WG, in order to help them move forward. They can still occasionally be brought up at meetings when it is deemed necessary.

The effectiveness of the Prioritization WG will be a direct consequence of our ability to draw the line between `P-critical` and `P-high` issues. There shouldn't be too many `P-critical` issues that compiler meetings become unmanageable, but critical issues shouldn't get lost in the list of `P-high` issues.